tests/integration/api/test_api.py

```diff
@@ -652,7 +653,7 @@ def test_monitor_muting(self):
         eq(dt, None)  # No response is expected.
 
         # We shouldn't be able to mute a simple alert on a scope.
-        assert_raises(ApiError, dog.Monitor.mute, monitor_id, scope='env:staging')
+        assert_raises(HTTPError, dog.Monitor.mute, monitor_id, scope='env:staging')
```

Comment:
The exception is proper to the API behavior so I think an ApiError makes more sense in this case.